orbit/cmd/orbit/orbit.go

```diff
@@ -462,8 +475,17 @@ func main() {
 					log.Info().Msg("No cert chain available. Relying on system store.")
 				}
 			}
+
 		}
 
+		_, nkr, e := enroll(fleetURL, enrollSecret, uuidStr, c.Bool("insecure"))
+		if e != nil {
+			log.Info().Msg("ERROR enrolling " + e.Error())
```

Comment:
Is execution meant to continue if we get an error here? If not, then consider doing `log.Fatal().Err(err).....`